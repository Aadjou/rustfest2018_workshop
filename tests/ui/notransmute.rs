fn main() {
    let footransmutebar = 42; //~ ERROR NOOOOO
}