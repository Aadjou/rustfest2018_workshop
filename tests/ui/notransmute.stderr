error: no. No. NO. NOOOOOO!!!! Like seriously, doesn't anyone read our coding guidelines?
 --> $DIR/notransmute.rs:2:9
  |
2 |     let footransmutebar = 42; //~ ERROR NOOOOO
  |         ^^^^^^^^^^^^^^^
  |
  = note: #[forbid(transmute)] on by default

error: aborting due to previous error

